app/javascript/src/components/category.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <div
+    class="category-container"
+  >
+    <div
+      class="category-main-product"
+    >
+      <div class="product-card">
+        <div class="product-image-container">
+          <product-image
+            :product="selectedProduct"
+            @click.native="clickAction"
+          />
+      </div>
+        <div class="choose-option-text">
+          Elige tu opción
+        </div>
+        <div
+          class="category-product-selector"
+        >
+          <div
+            v-for="(product, index) in category.products" :key="index"
```

Comment:
Attribute "bind" should be on a new line             vue/max-attributes-per-line